### PR TITLE
replace journal with glog

### PR DESCRIPTION
**Pull request by toanju**
current implementation of the logging facility leads to segfaults,
hereby it is replaced by glog

### CODE REVIEW

File: src/rofl/common/crofbase.cc [hunk 1]
```diff
@@ -622,8 +609,9 @@ void
 crofbase::handle_connect_refused(
 		crofctl& ctl, crofconn& conn)
 {
-	journal.log(LOG_INFO, "connection refused: ctlid=%s, auxid=%s, raddr=%s",
-			ctl.get_ctlid().str().c_str(), conn.get_auxid().str().c_str(), conn.get_raddr().str().c_str());
+	VLOG(2) << "connection refused: ctlid=" << ctl.get_ctlid().str().c_str()
+	   	<< ", auxid=" << conn.get_auxid().str().c_str() << ", raddr=" <<
```
Comment (DFritzsche) [inline src/rofl/common/crofbase.cc]: the `,` are not in line with other log msgs

File: src/rofl/common/crofconn.cc [hunk 2]
```diff
@@ -276,8 +267,7 @@ crofconn::set_state(
 		}
 
 	} catch (std::runtime_error& e) {
-
-		journal.log(LOG_RUNTIME_ERROR, "runtime error: %s", e.what()).set_caller(__PRETTY_FUNCTION__);
+		VLOG(1) << __PRETTY_FUNCTION__ << " runtime error: " << e.what();
```
Comment (DFritzsche) [inline src/rofl/common/crofconn.cc]: Shouldn't this be `LOG(ERROR)` ?

File: src/rofl/common/crofbase.cc (hunk 1 above)
Comment (toanju) [inline src/rofl/common/crofbase.cc]: just kept them as they were, but we should overall align the logging output separately from this

File: src/rofl/common/crofconn.cc (hunk 2 above)
Comment (toanju) [inline src/rofl/common/crofconn.cc]: in principle yes, but rofl by default never logged anything. That's why I just used the VLOGs. Basically ERROR -> VLOG(1), INFO -> VLOG(2) and TRACE -> VLOG(3) (should have added this to the commit message).

**Pull request merged**